src/jacobi.h

```diff
@@ -0,0 +1,32 @@
+#ifndef _JACOBI_LIB_
+#define _JACOBI_LIB_
+
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+
+#ifndef DIGITS_PRECISION
+    #define DIGITS_PRECISION 15
+#endif
+
+typedef double real_number;
+typedef unsigned int vec_size;
+
+#ifndef DIGITS_PRECISION
+    #define DIGITS_PRECISION 15
+#endif
```

Comment:
There are two defines for `DIGITS_PRECISION`. One should be removed
Also, do you think it should be a parameter passed to the function instead of being a define?

Reply:
I don't think it should be a parameter passed to the function, because this will not vary between multiple calls of the function so it would be redundant for the user to always put it when calling the function